bundle/src/main/java/org/cru/importer/GiveDataImportServlet.java

```diff
@@ -123,6 +124,14 @@ private boolean validateParams(SlingHttpServletRequest request, ParametersCollec
 				} else {
 					parametersCollector.setAcceptFilesPattern(acceptFilesPattern);
 				}
+	            Resource globalConfigs = getGlobalConfig(request, properties);
+	            if (globalConfigs == null) {
+	                resultsCollector.addError("globalConfigs configuration not found at configpath or node not exists at JCR.");
+	                isValid = false;
+	            } else {
+	                ValueMap globalProperties = globalConfigs.adaptTo(ValueMap.class);
+	                parametersCollector.setSanitizationMap(PropertiesUtil.toMap(globalProperties.get("sanitizationMap",String[].class), new String[]{}));
+	            }
```

Comment:
This spacing of this is off relative to the file's spacing, but I think that's true in several other places even before this commit, so maybe not worth fixing now. (Especially for non-long-term code.)